portal/src/locale-data/en.json

```diff
@@ -49,6 +49,7 @@
   "ScreenNav.anonymous-users": "Anonymous Users",
   "ScreenNav.single-sign-on": "Single-Sign On",
   "ScreenNav.passwords": "Passwords",
+  "ScreenNav.oauth-clients": "OAuth Clients",
```

Comment:
IIRC this should be changed to `Client Applications`.

Reply:
Also need to change title of the screen?

Reply:
Yes.